pkgs/development/tools/google-app-engine-go-sdk/default.nix

```diff
@@ -17,17 +19,17 @@ stdenv.mkDerivation rec {
         sha256 = "18hgl4wz3rhaklkwaxl8gm70h7l8k225f86da682kafawrr8zhv4";
       };
 
-  buildInputs = with python27Packages; [
-    (python27.withPackages(ps: [ cffi cryptography pyopenssl ]))
-  ];
+  buildInputs = [python27 makeWrapper];
 
   installPhase = ''
     mkdir -p $out/bin $out/share/
     cp -r "$src" "$out/share/go_appengine"
 
     # create wrappers with correct env
     for i in goapp appcfg.py; do
-      ln -s "$out/share/go_appengine/$i" "$out/bin/$i"
+      makeWrapper "$out/share/go_appengine/$i" "$out/bin/$i" \
```

Comment:
Have you read the [python](http://nixos.org/nixpkgs/manual/#python) section in the manual? `buildPythonApplication` might be useful here.